.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "03:00"
+    open-pull-requests-limit: 10
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "03:00"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "03:00"
+    open-pull-requests-limit: 10
```

Comment:
It'd be good to follow the process from: https://github.com/alphagov/govuk_publishing_components/blob/main/.github/dependabot.yml and just indicate the dependencies we want to vigilantly stay on top of (particularly as this repo is likely to have low levels of love).

I'd image we care about: 

Ruby:

Frameworks: rails, rspec-rails, capybara
Our own gems: rubocop-govuk (which we probably want to pin to a specific version to avoid frustration) and plek

Reply:
We talked this through and agreed we'd leave it as is, since this project's gemspec doesn't specify many dependencies so is unlikely to cause a dependabot-storm. For some gems, we don't expect (m)any updates anyway – e.g. `bootstrap-sass` and `jquery-rails`.

We can always change this in the future if it gets too noisy.